### PR TITLE
Adds persistence.existingClaim value to Chronograf chart

### PR DESCRIPTION
**Pull request by patrickjmcd**
In order to accommodate PersistentVolumeClaims that have already been created, I've added in the `persistence.existingClaim` value in [the same way that Grafana does it](https://github.com/grafana/helm-charts/blob/74624247a5676bd8d2c5460c47bd984aaad35712/charts/grafana/templates/_pod.tpl#L417).

This should allow users to use other storage methods (NFS in my case) to store Chrongraf configuration data.

I have tested this locally via `helm install...`

### CODE REVIEW

**Review by rawkode — APPROVED** [review]

**Review by rawkode — DISMISSED** [review]

File: charts/chronograf/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: chronograf
-version: 1.1.19
+version: 1.1.20
```
Comment (timhallinflux) [inline charts/chronograf/Chart.yaml:3]: just need to bump this forward.  We've had another merge since...

File: charts/chronograf/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
 appVersion: 1.8.8
```
Comment (timhallinflux) [inline charts/chronograf/Chart.yaml:4]: 1.8.9 is now out too.  Let's bump that forward as well.

File: charts/chronograf/Chart.yaml (hunk 1 above)
Comment (patrickjmcd) [reply charts/chronograf/Chart.yaml]: bumped in 9310ec2

File: charts/chronograf/Chart.yaml (hunk 2 above)
Comment (patrickjmcd) [reply charts/chronograf/Chart.yaml]: bumped in 9310ec2

**Review by timhallinflux — DISMISSED** [review]

**Comment by rawkode** [comment]
@patrickjmcd I think your recent commit has injected some weird formatting changes?

**Comment by rawkode** [comment]
No updates for a while. Please feel free to ping me if you'd like to work on this and get it merged.

**Pull request closed without merging**